doc/source/whatsnew/v0.23.4.txt

```diff
@@ -41,7 +41,7 @@ Bug Fixes
 
 **Indexing**
 
--
+- Fix OverflowError when trying to use 'inf' as na_value with int index column
```

Comment:
add the issue number here, use double-backticks around OverflowError. use double backticks on na_value; spell out int -> integer.